packages/@adobe/react-spectrum/test/ActionGroup.test.tsx

```diff
@@ -13,7 +13,7 @@
 import {ActionGroup, Item} from '../';
 import {Provider} from '@react-spectrum/provider';
 import React from 'react';
-import {render} from '@testing-library/react';
+import {render} from '@react-spectrum/test-utils';
```

Comment:
Re-exported all the testing library commands from `test-utils` now that we are overriding `render`, `renderHooks`, and `act`. Makes it easier IMO to just export from one place